CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 - Fix: support to lazy attributes and commands based in NGSIv2 (#104)
+- Fix: support to other lwm2m clients i.e. Leshan java client

```

Comment:
Please consider the following suggestion:

```suggestion
- Fix: support to lwm2m client that separe object instances using spaces
```

I think is more clear and more generic that way.

Reply:
accepted suggestion by 3591f6292c2516e65f743cf88db227abbbf51b77